deployment/hasura/metadata/databases/tables/actions/action_run.yaml

```
@@ -0,0 +1,47 @@
+table:
+  name: action_run
+  schema: actions
+configuration:
+  custom_name: "action_run"
+object_relationships:
+  - name: action_definition
```

Nonblocking: rename suggestion 
```suggestion
  - name: action
```

---

This broke some of the ui, so we reverted this change. Going to leave the relationship named `action_definition` for now.

---

Bumping as you left a 👍 but nothing was changed. Did you mean to commit a change or were you rejecting the suggestion?